run-tnf-container.sh

```diff
@@ -48,10 +49,13 @@ usage() {
 	  -k: set path to one or more local kubeconfigs, separated by a colon.
 	      The -k option takes precedence, overwriting the results of local kubeconfig autodiscovery.
 	      See the 'Kubeconfig lookup order' section below for more details.
+      -c: set path to one or more local dockercfgs, separated by a colon.
```

Comment:
Broken spacing.

Reply:
When looking at it in my editor, it's the same as line 49 (the -k section).

Reply:
Make sure:
 https://stackoverflow.com/questions/30140595/show-whitespace-characters-in-visual-studio-code